thrust/system/cuda/detail/cub/thread/thread_load.cuh

```diff
@@ -33,7 +33,7 @@
 
 #pragma once
 
-#include <cuda.h>
+//#include <cuda.h>
```

Comment:
Dead code?

Reply:
This is the CUDA backend. We do not modify that code to avoid future conflicts.